tests/sdbench.bats

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bats
+load test-helper
+
+@test "$clinom sdbench" {
+  run "${clicmd}" sdbench
+  assert_success && assert_output -p 'read' && assert_output -p 'write'
```

Comment:
This could also check for a normal numeric output for the read and write speeds on each line.

Sample output shows read and write speeds on 2 lines:
> read  - 45.5 MB/s
> write - 9.7 MB/s